projects.js

```diff
@@ -0,0 +1,114 @@
+    technologies: ['css', 'html', 'bootstrap', 'ruby'],
+    livedemo: 'https://gordonotieno.github.io/portfolio-website/',
+    sourceCode: 'https://github.com/GordonOtieno/portfolio-website/',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized readsno accounts or sign ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    image: './assets/images/Img-Placeholder.svg',
+    technologies: ['css', 'html', 'bootstrap', 'ruby'],
+    livedemo: 'https://gordonotieno.github.io/portfolio-website/',
+    sourceCode: 'https://github.com/GordonOtieno/portfolio-website/',
+  },
+];
+
+function getProjects(args) {
+  let html = '';
+  for (let i = 0; i < args.length; i += 1) {
+    html += ` <li class="project-list--li" tabindex="0">
+   <div class="project-detail">
+       <h3 class="project-title">${args[i].name}</h3>
+       <p class="project-desc">${args[i].description} </p>
+       <ul class="tech-list">
+           <li class="skills">${args[i].technologies[0]}</li>
+           <li class="skills">${args[i].technologies[1]}</li>
+           <li class="skills">${args[i].technologies[2]}</li>
+       </ul>
+   </div>
+       <button role="button" type="button" class="project_list--li__cta" data-toggle="modal" data-target="#myModal">See Project</button>
+</li>`;
+
+    document.querySelector('.work-section-container__project--list').innerHTML = html;
+  }
+}
+getProjects(projects);
+
+function displaymodal() {
+  let html = '';
+  html += ` <div class="modal fade" id="myModal" role="dialog">
+    <div class="modal-dialog">
+    
+      <!-- Modal content-->
+      <div class="modal-content">
+         <div class="modol-closing">
+          <button type="button" class="close" data-dismiss="modal">&times;</button> 
+         </div>
+        <div class="modal-top-content">
+          <h4 class="modal-header-title">Multi Post Stories</h4>
+        <div class="modal-body">
+          <ul class="tech-list model">
+          <li><button type="button">html</button></li>
+            <li><button type="button">bootstrap</button></li>
+            <li><button type="button">ruby</button></li>
+        </ul>
+        </div>
+        <div class="image-prag-container">
+          <div class="project-image">
+            <img src="./assets/images/SnapshootPortfolio.svg" alt="SnapshootPortfolio">
+          </div>
+          <div class="modol-pb-container">
+            <div class="modal-info">
+              Lorem Ipsum is simply dummy text of the printing 
+              and typesetting industry. Lorem Ipsum has been the industry's 
+              standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries,
+               but also the leap into electronic typesetting, remaining essent
+            </div>
```

Comment:
## Your Popup Window
![previous-reviewer](https://user-images.githubusercontent.com/93039359/180137202-6a5267aa-8546-47eb-b672-3e728de5a2c0.png)
## Figma Popup Window
![figma](https://user-images.githubusercontent.com/66230306/180040943-58a0432b-57ef-4509-bfd8-19ba938acc90.png)
- [ ] Great work on implementing the popup window but your popup is not very much similar to the Figma template :blush: Please make your popup window according to the Figma template :+1: You can use the example stylings i have done below :arrow_down:
### Styling Part 1
![styling-2](https://user-images.githubusercontent.com/93039359/180137035-7374bbdf-74af-4a7f-bdfa-604be0a95101.png)
### Styling Part 2
![styling-1](https://user-images.githubusercontent.com/93039359/180137052-8a335468-2413-4507-bc30-50c4fd0aed35.png)

**Note:** Implement both stylings to see the outcome :blush: